server/models/Tier.js

```diff
@@ -48,10 +51,15 @@ export default function(Sequelize, DataTypes) {
         type: DataTypes.STRING,
         allowNull: false,
         set(name) {
+          this.setDataValue('name', name);
+
           if (!this.getDataValue('slug')) {
-            this.slug = name;
+            // Try to generate the slug from the name. If it fails, for example if tier
+            // name is '😵️' we gracefully fallback on tier type
+            const slugFromName = slugify(name);
+            const slug = slugFromName || slugify(this.type || 'TIER');
```

Comment:
What happens if slug has a duplicate?

Reply:
There's no unique constraint on tier's slugs (we use the id in the URL to differentiate) so it should be fine